t/bulk.t

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use FindBin qw( $Bin );
+use lib qw( t/lib );
+use Test::More;
+use Sympa::Tools::File;
+use Sympa::Message;
+use ok 'Sympa::Bulk';
+use English;
+
+# during a test, syslog would be helpful as a note
+# so *Sympa::Log::syslog is diverted
+#
+# by default, syslog remains silent
+
+{ no warnings;
+    *Sympa::Log::syslog = 1 #  $ENV{SYMPA_TEST_SILENT_SYSLOG}
+    ? sub {}
+    : sub {
+        shift; # $self isn't expected anymore
+        my ( $level, $fmt, @params ) = @_;
+        note sprintf "syslog $level: $fmt", @params; }
+    ;
+}
+
```

Comment:
I feel reflection would be better to be used as little as possible, and initializing Sympa::Log instance with usual way (setting {log_to_stderr} will output logs to STDERR).

Reply:
i did that to make the syslog strings appears to the test report and i don't know if it's possible from the test framework itself. any idea to do that ?

Reply:
Either using `note()` or STDERR, output seem to be hidden by test harness.  So I didn't figure out why you needed large-scale gimmick to use former way (`note()`).

Reply:
There are test modules to capture logging. I think that defeats the principle of minimum of changes and is hard to read as well.